src/serviceconnector-passwordless/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+1.1.0
```

Comment:
Deleted args `key_vault_id` is a breaking change. Major version number should be increased to be 2.0.0.

For more info, please refer to the extension versioning doc: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#versioning-scheme